test/jdk/java/awt/Multiscreen/MultiScreenInsetsTest/MultiScreenInsetsTest.java

```diff
@@ -47,11 +47,6 @@ public class MultiScreenInsetsTest {
     private static final int SIZE = 100;
 
     public static void main(String[] args) throws InterruptedException {
-        if (!Platform.isLinux()) {
-            System.out.println("This test is for Linux only..." +
-                               "skipping!");
-            return;
-        }
 
         GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
         GraphicsDevice[] gds = ge.getScreenDevices();
```

Comment:
We throw `jtreg.SkippedException` if test cannot be run in the current environment.

You can find a sample usage in #22729.